jgo/jgo.py

```diff
@@ -265,7 +265,7 @@ def run_and_combine_outputs(command, *args):
 
 def find_endpoint(argv, shortcuts={}):
     # endpoint is first positional argument
-    pattern = re.compile(".*https?://.*")
+    pattern = re.compile("(.*https?://.*|\S:\\.*)")
```

Comment:
Is `\S` too general? I am not a Windows expert (especially the latest versions of Windows), but IIUC, you are limited to 26 drive letters A through Z. So better to use `[A-Z]` rather than `\S`, no?

Reply:
I think this is correct. I just pushed a change, to use `[a-zA-Z]` instead, since Windows paths aren't case sensitive. Thanks for taking a look at this!